tests-functional/pytest.ini

```diff
@@ -1,11 +1,14 @@
 [pytest]
-addopts = -s -v --tb=short
+addopts = -v --show-capture=all --junitxml=report.xml
 
 log_cli=true
-log_level=INFO
+log_level=DEBUG
+log_cli_level=DEBUG
 log_cli_format = %(asctime)s.%(msecs)03d %(levelname)s %(message)s
 log_cli_date_format = %Y-%m-%d %H:%M:%S
 
+junit_logging = all
+
```

Comment:
So if there is no obvious reason why we should not include debug, I propose to leave it like that, as it gives us a way more [context](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6697/16/testReport/tests.test_recover_mnemonic/TestRecoverMnemonic/test_recover_app_different_mnemonic_mnemonic_24_/)